modules/ROOT/pages/branded_desktop_client/branded_linux_desktop_clients.adoc

```diff
@@ -143,14 +143,16 @@ The url after unpacking is an internal minio url, or a non working example like
 You must change example.sh before it can be run.
 
 * Either
-   - replace the url with the correct url leading to the exact filder in which the archive was unpacked +
+   - replace the url with the correct url leading to the exact folder in which the archive was unpacked +
      (ending in `.../mycloud-2.10.0.6752-linux/` here).
 * Or 
    - Delete (or comment out) the line containing the url.
    - Make sure there is a `.repo-admin-prefix` file in the parent folder of the unpacked archive (here next to `mycloud-2.10.0.6752-linux`)
-   - Its contents is the base URL leading up to this parent folder. E.g. when the URL to the unpacked archive is `http://cloud.example.com/install/mycloud-2.10.0.6752-linux` then this command can be used to create the prefix file:
+   - Its contents is the base URL leading up to this parent folder. +
+     E.g. if the correct --url parameter were `http://cloud.example.com/install/mycloud-2.10.0.6752-linux/` +
+     then the prefix file can be created like this:
```

Comment:
> then the prefix file can be created like this:

I am missing the part coming after `this`...
Or did I miss something?

Reply:
"... created like this" refers to the next two items that follow below.